back-end/src/main/java/com/skynet/Skymed/model/Endereco.java

```diff
@@ -1,90 +1,110 @@
 package com.skynet.Skymed.model;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToMany;
 
+@Entity
 public class Endereco {
 
 	@Id
+	@Column(name = "end_iden")
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	private Long   end_id;
-	private String end_cod_ibge;
-	private String end_estado;
-	private String end_logradouro;
-	private String end_complemeto;
-	private String end_numero;
-	private String end_cep;
-	private String end_uf;
-	private String end_cidade;
-	
-	@OneToMany
-	private Pessoa pessoas;
-	
-	
-	public Pessoa getPessoas() {
-		return pessoas;
-	}
-	public void setPessoas(Pessoa pessoas) {
-		this.pessoas = pessoas;
-	}
-	public Long getEnd_id() {
-		return end_id;
+	private Long id;
+	@Column(name = "end_cod_ibge")
+	private String cod_ibge;
```

Comment:
```suggestion
	private String cod_ibge;
```

codigoIBGE por favor